larray/inout/excel.py

```diff
@@ -56,6 +55,96 @@ def read_excel(filepath, sheet=0, nb_axes=None, index_col=None, fill_value=np.na
         Engine to use to read the Excel file. If None (default), it will use 'xlwings' by default if the module is
         installed and relies on Pandas default reader otherwise.
     **kwargs
+
+    Returns
+    -------
+    LArray
+
+    Examples
+    --------
+    >>> import os
+    >>> from larray import EXAMPLE_FILES_DIR
+    >>> fname = os.path.join(EXAMPLE_FILES_DIR, 'examples.xlsx')
+
+    Read array from first sheet
+
+    >>> read_excel(fname)
+    a  a0  a1  a2
+        0   1   2
+
+    Read array from a specific sheet
+
+    >>> read_excel(fname, '2d')
+    a\\b  b0  b1
+      1   0   1
+      2   2   3
+      3   4   5
+
+    Missing label combinations
+
+    >>> # let's take a look inside the sheet 'missing_values'.
+    >>> # they are missing label combinations: (a=2, b=b0) and (a=3, b=b1):
+
+    a  b\c  c0  c1  c2
+    1  b0   0   1   2
+    1  b1   3   4   5
+    2  b1   9   10  11
+    3  b0   12  13  14
+
+    >>> # by default, cells associated with missing label combinations are filled with NaN.
+    >>> # In that case, an int array is converted to a float array.
+    >>> read_excel(fname, 'missing_values')
```

Comment:
I **_wonder_** if the combination of this sheet name and the fact the argument ("sheet") is used positionally is not potentially confusing. Maybe using sheet='missing_values', or renaming the sheet to something like 'with_missings' would make it more obvious.

The same question applies to HDF array names.